message ReqGet
{
	required string imageID = 1;
	required int32	width = 2;
	required int32 height = 3;
}

message ReqSet
{
	required string imageName = 1;
	required string imageBuff = 2;
}

enum RspCode
{
	REQ_FAILED = 0;
	REQ_SUCCESS = 1;
}

enum ErrCode
{
	ERR_FILE_NOT_FOUND = 0;
	ERR_INVALID_PARAMS = 1;
}

message ReqResponse{
	required RspCode rspCode = 1;
	optional ErrCode errCode = 2;

	optional string newname = 3;
}